### ISSUE TITLE
kernel size in  class FBnetblock

### ISSUE THREAD

**Issue by 666zz666** [JunrQ/NAS#8]
You didn't use kernel size in class FBnetblock.
What's wrong with Bn?

**Comment by JunrQ** [comment]
kernel size is specified in fbnet-pytorch/blocks.py by parameter `_kernel`
and BN will help traing deep neural network

**Comment by 666zz666** [comment]
https://github.com/JunrQ/NAS/blob/2cdf88b8a4943f8ada6ce7feb38669aa05331516/fbnet-pytorch/blocks.py#L43
https://github.com/JunrQ/NAS/blob/2cdf88b8a4943f8ada6ce7feb38669aa05331516/fbnet-pytorch/blocks.py#L55
It's three,not kernel-size.

**Comment by JunrQ** [comment]
You are right,that’s a bug.

**Comment by 666zz666** [comment]
So,Why don't you use bn in your code?

**Comment by JunrQ** [comment]
The paper doesn't use bn

**Comment by 666zz666** [comment]
Maybe this is the default

**Comment by JunrQ** [comment]
maybe, BN is the default for deep neural network.

The insteresting thing is that the search part converges very well without BN.
But training the sampled model without BN will fail to converge.

Any idea why this happens?

**Comment by GuntherZhong** [comment]
> You are right,that’s a bug.

Hi, it seems that the padding size should be modified too :-)

**Comment by 666zz666** [comment]
Maybe padding = (kernel_size - 1) // 2 is better.
